tutorials/export/exporting_pcks.rst

```diff
@@ -51,6 +60,23 @@ PCK files usually contain, but are not limited to:
 The PCK files can even be an entirely different Godot project, which the
 original game loads in at runtime.
 
+.. _doc_exporting_pcks_about_mods_using_pck_files:
+
```

Comment:
```suggestion

.. UPDATE: Recommendation may change. If we ever implement sandboxing or another solution
.. change this section.

```
Recently we started using these kind of comments to keep track of information in the docs that is inherently likely to change.